pageserver/src/http/openapi_spec.yml

```diff
@@ -669,6 +669,13 @@ paths:
         Detach a timeline from its ancestor and reparent all ancestors timelines with lower `ancestor_lsn`.
         Current implementation might not be retryable across failure cases, but will be enhanced in future.
         Detaching should be expected to be expensive operation. Timeouts should be retried.
+      parameters:
+        - name: detach_behavior
```

Comment:
Other parameters (`tenant_shard_id`/`timeline_id`) are declared above, I suggest moving this one there too, so all parameters would be declared in one place

Reply:
I believe these are different parameters. In open API you could add parameters to both places. The upper place applies to all HTTP methods. Like, you have a path and then you could have post,put,get. The parameters listed on the path apply to all methods. While here I want this parameter to be specific to `put`. So, I think it's reasonable to have it here?

Reply:
It looks strange to me because there is only one method - `put`, but I'm not familiar with open API and I'm not going to insist on it

I think it's not important, if you separated these parameters intentionally - let's just merge

Reply:
okay, merging